pkgs/native_assets_cli/lib/src/utils/yaml.dart

```diff
@@ -16,3 +16,18 @@ String yamlEncode(Object yamlEncoding) {
   );
   return editor.toString();
 }
+
+T as<T>(Object? object) {
+  if (object is T) {
+    return object;
+  }
+  throw FormatException(
+    "Unexpected value '$object' in YAML. Expected a $T.",
+  );
+}
+
+extension YamlMapCast on YamlMap {
+  Map<K, V> formatCast<K, V>() => <K, V>{
```

Comment:
Why not also call this `as` - as it also does `as` on the individual `K` & `V`?

Reply:
That's what I did at first, but the `as` in the implementation starts resolving to it. 🙈